request.js

```diff
@@ -1364,6 +1364,8 @@ Request.prototype.aws = function (opts, now) {
       host: self.uri.host,
       path: self.uri.path,
       method: self.method,
+      service: opts.service,
```

Comment:
Is it expected that custom domains would only be available for version 4 signing?